guides/hack/11-built-in-types/22-arrays-examples/assignment-with-value-containers.php

```diff
@@ -0,0 +1,30 @@
+<?hh // strict
+
+namespace Hack\UserDocumentation\Types\Arrays\Examples\AssignmentWithValueContainers;
+
+use namespace HH\Lib\Str;
+
+<<__EntryPoint>>
+function main(): void {
+    require_once __DIR__.'/../../../../vendor/autoload.hack';
+    \Facebook\AutoloadMap\initialize();
```

Comment:
This should be unneeded - the test runner automatically pulls this in

Reply:
So, my `<<__EntryPoint>>` isn't the _real_ entrypoint.
<img src="https://user-images.githubusercontent.com/31805625/76356472-4cf86900-6316-11ea-9810-ccf318430bff.png" width=175/>

Reply:
well... kinda - `auto_prepend_file` ini setting is still a thing :p